config/opendatahub-io/org.yaml

```diff
@@ -394,6 +394,7 @@ orgs:
         maintainers:
         - jkoehler-redhat
         members:
+        - andrewballantyne
```

Comment:
Oh -- this needs a new commit

Not sure why I wasn't in this list 🤔

Reply:
You already had higher level access through the `ODH Dashboard Owners` team

Reply:
Odd not to double down, no? The two groups are not "related" per se.

I can remove this if you feel it has no value...

Reply:
Actually, ignore my previous statement.  At one time you had `Admin` access to the odh-dashboard repo as an idividual but I think it was removed without moving you to the team with the same access level.